drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -289,13 +292,17 @@ void RasterizerGLES3::_blit_render_target_to_screen(RID p_render_target, Display
 
 	// TODO: do we need a keep 3d linear option?
 
+	// make sure we are drawing to the right context
+	DisplayServer::get_singleton()->gl_window_make_current(p_screen);
+
 	if (rt->external.fbo != 0) {
 		glBindFramebuffer(GL_READ_FRAMEBUFFER, rt->external.fbo);
 	} else {
 		glBindFramebuffer(GL_READ_FRAMEBUFFER, rt->fbo);
 	}
 	glReadBuffer(GL_COLOR_ATTACHMENT0);
 	glBindFramebuffer(GL_DRAW_FRAMEBUFFER, GLES3::TextureStorage::system_fbo);
+	// NOTE: flips content upside down to correct for coordinates
```

Comment:
IMO any comment is a "note" (unless it's an explicit TODO/FIXME/WARNING), so it's redundant to specify it. I think this would be just as fine:
```suggestion
	// Flip content upside down to correct for coordinates.
```

Reply:
You can do the changes locally and amend the commit, then force push.

Alternatively if there are lot of suggestions which you want to accept, you can do it with GitHub but then you'll indeed need to squash the extra commit (and possibly drop me as co-author because that's not warranted for style nitpicks :)).